epistles-v1/original/14-epistles.adoc

```diff
@@ -1170,6 +1178,7 @@ which gathers to God.
 [.signed-section-signature]
```

Comment:
Above here ^^ line 1083 (`So let this be read in all your men...`) -- let's make that a Postscript as well.